### PR TITLE
Bump citus version to 7.4devel

### PR DESCRIPTION
**Pull request by velioglu**

### CODE REVIEW

**Comment by metdos** [comment]
@velioglu why are we waiting for this PR?

**Comment by velioglu** [comment]
I can rebase and merge it after a review.

File: configure [hunk 1]
```diff
@@ -661,7 +661,6 @@ infodir
 docdir
 oldincludedir
 includedir
-runstatedir
```
Comment (metdos) [inline configure]: I remember that we have talked about this and you said we removed this 'runstatedir' on purpose?

Comment (metdos) [reply configure]: Apart from this question/clarification, it looks good 🚢

Comment (velioglu) [reply configure]: Yes, it was emerged due to running autoconf on Ubuntu and should be removed before merging, so we are removing it with this PR.

**Comment by jasonmp85** [comment]
Can you also update `citus_version.h` inside the `windows` folder?

**Pull request merged**